boards/nucleo-l452re/include/periph_conf.h

```diff
@@ -0,0 +1,230 @@
+/*
+ * Copyright (C) 2017 Freie Universität Berlin
+ *               2017 Inria
+ *               2017 HAW-Hamburg
+ *               2018 Fundacion Inria Chile
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_nucleo-l452re
```

Comment:
There's a problem here (just noticed that sorry): this doxygen group is never defined. Have a look [here](https://github.com/RIOT-OS/RIOT/blob/master/boards/nucleo-l476/include/periph_conf.h#L12) for an example.

After this is fixed, we are good.